src/components/cards/comparison/ChangeItem.vue

```diff
@@ -1,28 +1,61 @@
 <!-- Visualizes a delta between two controls -->
 <template>
   <v-row>
-    <v-col class="pa-0" cols="1">
-      <v-card class="pa-2" outlined tile>
-        <slot name="name" />
-      </v-card>
-    </v-col>
-    <v-col class="pa-0" cols="5">
-      <v-card class="pa-2" outlined tile>
-        <slot name="old" />
-      </v-card>
-    </v-col>
-    <v-col class="pa-1" cols="1" align-self="center">
-      <v-icon> arrow_forward </v-icon>
+    <v-col cols="3" xs="3" sm="2" md="2" lg="1" xl="1" class="pa-0">
+      <slot name="name" />
     </v-col>
-    <v-col class="pa-0" cols="5">
-      <v-card class="pa-2" outlined tile>
-        <slot name="new" />
+    <v-col
+      class="pa-0"
+      v-for="(value, i) in values"
+      :key="i"
+      cols="4"
+      xs="4"
+      sm="3"
+      md="3"
+      lg="3"
+      xl="3"
```

Comment:
This shouldn't be necessary, if a value isn't set for a breakpoint then it just inherits from the one before it. The following is equivalent:

```ruby
    <v-col
      class="pa-0"
      v-for="(value, i) in values"
      :key="i"
      cols="4"
      sm="3"
>
```